lib/db/api.js

```diff
@@ -1611,6 +1610,16 @@ dbapi.getInstalledApplications = function(message) {
   return db.run(r.table('devices').get(message.serial))
 }
 
+dbapi.setDeviceGroupOwner = function(message) {
+  return db.run(r.table('devices').get(message.serial).update({
+    group: {
+      owner: {
+        email: process.env.STF_ADMIN_EMAIL,
```

Comment:
we don't have .env for the agent servers... it is available only for stf-app etc services... registering admin on every agent seems overcompilated. so we have to find a way to get this info from rehinkdb I guess with init/reinit on the stf-app startup(?!)

Reply:
What about [this approach?](https://github.com/zebrunner/stf/pull/859/commits/ae403d24b38603f3b9029690ab151236df787d0f#diff-2d7d29645cdcf64cae927e6ef11e3e44c09de2920d0b40e48f0f4f5adf39f2a0)?

In other words: 

1. Make request to a new STF API endpoint
2. API uses `dbapi` method
3. Method returns owner data
4. Agent server uses owner data

Reply:
Yep, it should work. Make sure case with updating current owner on restart is supported